src/main/java/com/redislabs/redisgraph/Utils.java

```diff
@@ -0,0 +1,24 @@
+package com.redislabs.redisgraph;
+
+/**
+ * Utilities class
+ */
+public class Utils {
+
+    /**
+     *
+     * @param str - a string
+     * @return the input string surounded with quotation marks, if needed
+     */
+    public static String quoteString(String str){
+        StringBuilder sb = new StringBuilder();
```

Comment:
1.  In case the string is already quoted you should avoid creating any string or StringBuilder
2. In case you do need to create new StringBuilder(str+2)